Aztec/Classes/Converters/HTMLNodeToNSAttributedString.swift

```diff
@@ -240,6 +249,22 @@ class HMTLNodeToNSAttributedString: SafeConverter {
         .h3: HeaderFormatter(headerLevel: .h3),
         .h4: HeaderFormatter(headerLevel: .h4),
         .h5: HeaderFormatter(headerLevel: .h5),
-        .h6: HeaderFormatter(headerLevel: .h6)
+        .h6: HeaderFormatter(headerLevel: .h6),
+        .span: ColorFormatter()
```

Comment:
Humm, good catch, I think I will change this a bit and made styles be parsed in a different way. 

Ideally they can be applied to any element so I will check styles in a dedicated pass just for the style attribute in elements. 
 
I will update the PR.